libraries/app/application.cpp

```diff
@@ -160,6 +160,27 @@ void application_impl::reset_p2p_node(const fc::path& data_dir)
       _p2p_network->add_seed_nodes(seeds);
    }
 
+   if ( _options->count( "disable-peer-advertising" ) )
+   {
+      if ( _options->at( "disable-peer-advertisng").as<bool>() )
+      {
+         _p2p_network->disable_peer_advertising();
+      }
+   } 
+   else
+   {
+      if( _options->count( "advertise-peer-algorithm" ) )
+      {
+         std::string algo = _options->at("advertise-peer-algorithm").as<string>();
+         fc::optional<std::vector<std::string>> list;
+         if ( algo == "list" && _options->count("advertise-list") )
```

Comment:
`advertise-list` is not in parameter list. Same for `exclude-list`. (maybe typo)

Reply:
Yes, typos. Found several. Fixing.